extensions/amp-story/1.0/amp-story.css

```diff
@@ -260,15 +259,17 @@ amp-story-page[distance="2"] {
   /* Prevent someone from making this a full-screen image */
   background-image: none !important;
   background-color: #000 !important;
-  z-index: 100001 !important;
-  animation: i-amphtml-bookend-overlay-opacity 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) forwards !important;
+  visibility: visible;
+  z-index: 100003 !important;
+  transition: visibility 0.3s, opacity 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
 }
 
 .i-amphtml-story-opacity-mask[hidden] {
   opacity: 0 !important;
   pointer-events: none !important;
   display: block !important;
-  animation: i-amphtml-bookend-overlay-opacity 0.2s cubic-bezier(0.4, 0.0, 1, 1) reverse !important;
+  visibility: hidden;
+  transition: visibility 0.2s, opacity 0.2s cubic-bezier(0.4, 0.0, 1, 1) !important;
```

Comment:
This looks to me like it shouldn't work, since the `visibility: hidden` is applied immediately when the transition begins. Does the transition animation still happen?

Reply:
Haha yes! I was poking around a lot and read [this documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#Interpolation) that made it make sense. Namely that `Visibility values are interpolable between visible and not-visible.`.

Reply:
wow nice TIL

Reply:
GitHub needs a `:exploding_head:` reaction